release_notes/ocp-4-17-release-notes.adoc

```diff
@@ -1113,7 +1113,7 @@ In this release, if your cluster is installed on a bare metal platform, you can
 [id="ocp-4-17-support-for-compute-nodes-on-amd-cpus_{context}"]
 ==== Support for compute nodes with AMD EPYC Zen 4 CPUs
 
-From release 4.17.10, you can use PerformanceProfiles to configure compute nodes on machines equipped with AMD EPYC Zen 4 CPUs such as Genoa and Bergamo. Per-pod power management is currently not supported on AMD.
+From release 4.17.10, you can use PerformanceProfiles to configure compute nodes on machines equipped with AMD EPYC Zen 4 CPUs, such as Genoa and Bergamo. Only single NUMA domain (NPS=1) configurations are supported. Per-pod power management is currently not supported on AMD.
```

Comment:
```suggestion
From release 4.17.10, you can use the `PerformanceProfiles` custom resource (CR) to configure compute nodes on machines equipped with AMD EPYC Zen 4 CPUs, such as Genoa and Bergamo. Only single NUMA domain (NPS=1) configurations are supported. Per-pod power management is currently not supported on AMD.
```

A nit: Adding back ticks as this is an [object name](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#object-references).

 > An object reference is when you are referring to the actual instance of an API object, where the object name is important.